readme.md

```diff
@@ -2,28 +2,44 @@
 
 ## Installation
 
+#### Prerequisites:
+Ensure you have the following installed first: 
+- librocksdb-dev
+- libclang-dev
+- clang lldb lld
+- build-essential 
+
+You will also need  rust and nightly installed. 
+To install Rust:
+```
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+To install nightly:
+```
+rustup install nightly-2021-06-09
+```
 ### Node
+Navigate to substrate node:
 
-You will need a specific version of rust nightly in order to compile:
-
-`rustup install nightly`
-
-Wasm toolchain:
-
-`rustup target add wasm32-unknown-unknown --toolchain nightly`
+```
+cd substrate-node
+```
 
-Now you can build.
+Add Wasm toolchain:
+```
+rustup target add wasm32-unknown-unknown --toolchain nightly-2021-06-09
+```
 
-```sh
-cd substrate-node
-# make sure you run nightly
-rustup override set nightly
-cargo build
+Now you can build:
+```
+cargo +nightly-2021-06-09 build --release
 ```
 
-This will build the node binary in debug mode, once built you can execute it by doing following:
+This will build the node binary in release mode, once built you can execute it by doing following:
 
-`./target/debug/tfchain --dev --ws-external`
+```
+./target/release/tfchain --ws-external --rpc-methods Unsafe --dev
```

Comment:
`--rpc-methods Unsafe` is optional, no need to add this as the default run command